stub/src/test/java/io/grpc/stub/ClientCallsTest.java

```diff
@@ -203,6 +220,48 @@ public void blockingUnaryCall2_interruptedWaitsForOnClose() throws Exception {
     assertTrue("context not cancelled", methodImpl.observer.isCancelled());
   }
 
+  @Test
+  public void blockingUnaryCall_HasBlockingStubType() {
+    when(mockChannel.newCall(
+        ArgumentMatchers.<MethodDescriptor<Integer, Integer>>any(), any(CallOptions.class)))
+        .thenReturn(mockClientCall);
+    // we only interested in the call options, stop the call by thrown.
+    doThrow(new RuntimeException("intended")).when(mockClientCall).sendMessage(any(Integer.class));
```

Comment:
`unaryBlockingCallFailed()` uses a `NoopClientCall` that calls `listener.onClose()` in its `start()`. That is closer to the reality. We should also keep the same style throughout a file.

Reply:
Done.